### PR TITLE
Update playbooks for 6.1

### PR DESCRIPTION
**Pull request by justinegeffen**
Updates to playbooks documentation:
- Preview section
- Notifications section

@stephenvanhemmen, could you take a look at the content let me know your feedback? Thank you! :)

### CODE REVIEW

File: source/playbooks/notifications-and-updates.rst [hunk 1]
```diff
@@ -6,5 +6,11 @@ To help you keep track of your runs, tasks, and statuses a daily digest is sent
 Daily digest
 ------------
 
+When you're running playbooks in multiple channels it can be overwhelming. The daily digest is sent once a day, in the morning. It lists the actionable items for each run, as well as any outstanding tasks or status updates required. Select the run name to move to that channel.
+
+The digest is on by default. To turn it off, use the slash command ``/playbook digest``.
```
Comment (stephenvanhemmen) [inline source/playbooks/notifications-and-updates.rst:11]: The slash command is `/playbook settings digest off` (admittedly a bit of a mouthful but works the same way as Github's)

File: source/playbooks/notifications-and-updates.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+=========================
+
+To help you keep track of your runs, tasks, and statuses a daily digest is sent via Direct Message in Mattermost.
+
+Daily digest
+------------
+
+When you're running playbooks in multiple channels it can be overwhelming. The daily digest is sent once a day, in the morning. It lists the actionable items for each run, as well as any outstanding tasks or status updates required. Select the run name to move to that channel.
+
+The digest is on by default. To turn it off, use the slash command ``/playbook digest``.
```
Comment (justinegeffen) [inline source/playbooks/notifications-and-updates.rst:11]:
```suggestion
The digest is on by default. To turn it off, use the slash command ``/playbook digest off``.
```

File: source/playbooks/notifications-and-updates.rst (hunk 1 above)
Comment (justinegeffen) [reply source/playbooks/notifications-and-updates.rst]: Done!

**Review by stephenvanhemmen — APPROVED** [review]
Looks great! 🎉

File: source/playbooks/notifications-and-updates.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Notifications and Updates
+=========================
+
+To help you keep track of your runs, tasks, and statuses a daily digest is sent via Direct Message in Mattermost.
```
Comment (cwarnermm) [inline source/playbooks/notifications-and-updates.rst:4]:
```suggestion
To help you keep track of your runs, tasks, and statuses, a daily digest is sent via Direct Message in Mattermost.
```

File: source/playbooks/notifications-and-updates.rst [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+Daily digest
+------------
+
+When you're running playbooks in multiple channels it can be overwhelming. The daily digest is sent once a day, in the morning. It lists the actionable items for each run, as well as any outstanding tasks or status updates required. Select the run name to move to that channel.
```
Comment (cwarnermm) [inline source/playbooks/notifications-and-updates.rst:9]:
```suggestion
Running playbooks in multiple channels can be overwhelming. The daily digest is sent once a day, in the morning. It lists the actionable items for each run, as well as any outstanding tasks or status updates required. Select the run name to move to that channel.
```

File: source/playbooks/notifications-and-updates.rst [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+The digest is on by default. To turn it off, use the slash command ``/playbook digest off``.
```
Comment (cwarnermm) [inline source/playbooks/notifications-and-updates.rst:11]: Should this be ``/playbook settings digest off``?

**Review by cwarnermm — COMMENTED** [review]
Good update! I've provided feedback inline.

File: source/playbooks/notifications-and-updates.rst [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+=========================
+
+To help you keep track of your runs, tasks, and statuses, a daily digest is sent via Direct Message in Mattermost.
+
+Daily digest
+------------
+
+Running playbooks in multiple channels can be overwhelming. The daily digest is sent once a day, in the morning. It lists the actionable items for each run, as well as any outstanding tasks or status updates required. Select the run name to move to that channel.
+
+The digest is on by default. To turn it off, use the slash command ``/playbook digest off``.
```
Comment (justinegeffen) [inline source/playbooks/notifications-and-updates.rst:11]:
```suggestion
The digest is on by default. To turn it off, use the slash command ``/playbook settings digest off``.
```

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Review by cpoile — APPROVED** [review]
Thanks @justinegeffen !

**Pull request merged**